packages/eslint-plugin-twenty-ts/src/tests/all.spec.test.ts

```diff
@@ -0,0 +1,90 @@
+import { RuleTester } from "@typescript-eslint/rule-tester";
```

Comment:
let's rename this file effect-component.spec.ts

I've also notice that test files are inconsistently named. They should all be suffixed with .spec.ts :)

Reply:
done